Makefile

```diff
@@ -8,7 +8,7 @@ VERSION_FILE ?= VERSION-DEV
 include $(VERSION_FILE)
 
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true"
+CRD_OPTIONS ?= "crd:trivialVersions=true,crdVersions=v1"
```

Comment:
How far behind, the generated CRDs should be supported? crdVersions v1beta1 is still needed for kubernetes < 1.16

Reply:
I think it's up to the user to generate the `v1beta1` version of the application CRD.
I added a comment to show how to set the CRD_OPTIONS for the `v1beta1` version.